### PR TITLE
Add dependency-section into doc

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -10,24 +10,48 @@ Additionally, the PR needs to be approved by one of the core project members
...
 ```
-$ go test ./...
-?   	github.com/openshift/ci-tools	[no test files]
-ok  	github.com/openshift/ci-tools/cmd/ci-operator	0.036s
-ok  	github.com/openshift/ci-tools/pkg/api	0.029s
-?   	github.com/openshift/ci-tools/pkg/interrupt	[no test files]
-?   	github.com/openshift/ci-tools/pkg/junit	[no test files]
-ok  	github.com/openshift/ci-tools/pkg/steps	0.044s
+
+TODO: Figure out the error when
```
Comment (stevekuznetsov) [inline CONTRIBUTORS.md]: want to remove this and below?

File: Makefile [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 all: lint test build
 .PHONY: all
 
 build:
 	go build ./cmd/...
 .PHONY: build
 
+build_folder := ./_output
+
+build-bin:
```
Comment (stevekuznetsov) [inline Makefile]: Why do we want this?

File: CONTRIBUTORS.md (hunk 1 above)
Comment (hongkailiu) [reply CONTRIBUTORS.md]: Removed.

File: Makefile (hunk 2 above)
Comment (hongkailiu) [reply Makefile]: Removed.

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -10,24 +10,36 @@ Additionally, the PR needs to be approved by one of the core project members
...
 ## Build
 
-To obtain sources and build the `ci-operator` binary, use `go get` and `go build`:
+To obtain sources and build the binaries provided by this repo, run the following commands:
 
 ```
 $ go get github.com/openshift/ci-tools
 $ cd ${GOPATH}/src/github.com/openshift/ci-tools
-$ go build ./cmd/ci-operator
+$ make install
```
Comment (droslean) [inline CONTRIBUTORS.md]: well, we are using `go build ./cmd/ci-operator` to actually build the images in the parent directory in order to use them in the `Dockerfiles`. So `make bin-build` or just `make build` makes more sense here.

**Pull request merged**